### PR TITLE
Additional style class selectors

### PR DESCRIPTION
**Pull request by SteveBrandt**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -11,6 +11,12 @@ var defaultOptions = {
     objectName   :   'Css'
 };
 
+var selectors = [];
```
Comment (0x6ee3) [inline index.js]: 🔥 State! 🔥

**Review by marttinen — CHANGES_REQUESTED** [review]

File: index.js (hunk 1 above)
Comment (0x6ee3) [reply index.js]: I don't get the reason behind `selectors` and `resetSelectors`...

Comment (0x6ee3) [reply index.js]: Ok, I get it now. Its for supporting multiple files. But its still not a pretty solution... 🤔

Comment (SteveBrandt) [reply index.js]: I'll think about it

**Pull request merged**